automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -54,7 +54,7 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
             // user installed packages should be in directory where PD is mounted
             val getFuzzyWuzzyLocation = "! pip3 show fuzzywuzzy"
             notebookPage.executeCell(getFuzzyWuzzyLocation, cellNumberOpt = Some(1)).get should include(
-              "/home/jupyter/notebooks/packages"
+              "/home/jupyter/packages"
```

Comment:
I believe this would not be `/packages` - we added that ourselves to give packages their own directory to live in so they wouldn't crowd the `/notebooks` dir. I think what python installs to is `/home/jupyter/.local/bin`

Reply:
are you thinking PIP_TARGET is updated?